src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
+import Display from './components/Display';
+import ButtonPanel from './components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
```

Comment:
Use a fragment instead of this <div className="App"> like you have done in other components.